selenium/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/SeleniumWebDriver.java

```diff
@@ -334,4 +349,36 @@ public void switchFromDashboardIframeToIde(int timeout) {
   private WebDriverWait wait(int timeOutInSeconds) {
     return new WebDriverWait(this, timeOutInSeconds);
   }
+
+  public String getGridNodeContainerId() throws IOException {
+    if (!gridMode) {
+      throw new UnsupportedOperationException("We can't get grid node container id in local mode.");
+    }
+
+    if (gridNodeContainerId == null) {
+      String getGridNodeInfoUrl =
+          format("http://localhost:4444/grid/api/testsession?session=%s", getSessionId());
+
```

Comment:
What about id grid will be have other configuration in some specific cases? Not `localhost:4444`

Reply:
Our tests are actually [stick to "localhost"](https://github.com/eclipse/che/blob/master/selenium/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/SeleniumWebDriver.java#L89), but port can be changed. So, I should definitely use `webDriverPort` value instead of 4444.

Thank you for note.